### DISCUSSION TITLE
Split vertical slopes sprite group

### DISCUSSION THREAD

**Discussion by spacek531** [OpenRCT2/OpenRCT2#17004] [Feature requests] [answered]
It would be useful if the vertical slopes sprite group was split into three sub sprite groups:
- Vertical slopes (75 and 90 degrees)
- Loop slopes (105 to 165 degrees)
- Inverted (180 degrees)

Separating the first is useful for my RCT1 project as some vehicles do not have beyond-vertical sprites. Separating the second is useful for my custom rides and my RCT1 project because some rides feature inversions without featuring loops (ultra twist coaster, future custom rides).

Importing dat files properly into this new scheme is no issue. Importing json will be some issue, unless an alternate name for vertical slopes can be found.

**Comment by duncanspumpkin** [comment, marked as answer]
This has been implemented by yourself now, no?

**Reply by spacek531** [reply]
yes
